meshroom/core/node.py

```diff
@@ -728,6 +730,14 @@ def clearData(self):
             shutil.rmtree(self.internalFolder)
             self.updateStatusFromCache()
 
+    @Slot(result=str)
+    def getStartDateTime(self):
+        """ Return the date (str) of the first running chunk """
+        if not self.isAlreadySubmittedOrFinished() or len(self._chunks) == 0:
```

Comment:
Submitted has no start time.
Status that can have a start time:
``` 
 43 class Status(Enum):
  44     """
  45     """
  46     NONE = 0 => no
  47     SUBMITTED => no
  48     RUNNING => yes
  49     ERROR => yes
  50     STOPPED => yes
  51     KILLED => yes
  52     SUCCESS => yes
```

Reply:
Maybe remove the status check on the node itself and add it on the loop over chunks.